.editorconfig

```diff
@@ -9,3 +9,6 @@ insert_final_newline = true
 
 [*.md]
 trim_trailing_whitespace = false
+
+[.rs]
```

Comment:
this might need a `*`. generally i like to avoid setting up & checking in editor related stuff as it fails to live up to the promise. maybe we can consider removing it altogether?

Reply:
I've removed the rs part and changed to `*` to `*.js`